docs/source/user_guide/plots.rst

```diff
@@ -860,7 +860,7 @@ Plotting backend
 ----------------
 
 Most plotting methods in the
-:meth:`fiftyone.core.plots <fiftyone.core.plots.base>` module provide an
+:meth:`fiftyone.core.plots <fiftyone.core.plots.base>` module provides an
```

Comment:
`provide` is actually correct here

Reply:
Thanks, Brian, I reverted it back to the original now